app/upw/index.js

```diff
@@ -12,6 +12,14 @@ router.get('*', (req, res, next) => {
   next()
 })
 
+router.get('*', (req, res, next) => {
```

Comment:
Our CSP prevents us simply using the `referrer` header, so we'e storing the `previousUrl` with this piece of middleware. It's a naive solution that works for now, i.e. single pages branching off the main FormWizard workflow

Reply:
I think that's fine though - it's as complex as it needs to be (i.e. not very!)

Reply:
😅